adhocracy4/modules/migrations/0003_rm_unique_name.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-08-10 09:57
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('a4modules', '0002_default_ordering_weight'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='module',
+            name='description',
+            field=models.CharField(blank=True, help_text='This short description will appear on the header of the module and project detail pages. It should briefly state the goal of the module in max. 250 chars.', max_length=250, null=True, verbose_name='Short description of the module'),
+        ),
+        migrations.AlterField(
+            model_name='module',
+            name='name',
+            field=models.CharField(help_text='This title will appear in the timeline and the header on the module and project detail pages. It should be max. 512 characters long', max_length=512, verbose_name='Title of the module'),
```

Comment:
good point. i was thinking from the meinberlin/product perspective. but i am not sure if the (new) timeline should be in core. i guess it's to specific for our different projects. i think we could remove the helptext.